src/runtime/hexagon/hexagon_device_api.h

```diff
@@ -190,16 +194,15 @@ class HexagonDeviceAPI final : public DeviceAPI {
            (DLDeviceType(dev.device_type) == kDLCPU);
   }
 
-  //! \brief Manages underlying HexagonBuffer allocations
+  //! \brief Manages RPC HexagonBuffer allocations
+  // rpc_hexbuffs is used only in Alloc/FreeRpcDataSpace
+  std::unique_ptr<HexagonBufferManager> rpc_hexbuffs;
```

Comment:
Since `rpc_hexbuffs` is initialized in the constructor and is never freed, does it need to be a `std::unique_ptr<HexagonBufferManager>` instead of `HexagonBufferManager`?

Reply:
It could be static, yes.  I'll make that change.